Public/Src/Deployment/privatePackages.dsc

```diff
@@ -80,8 +80,6 @@ namespace PrivatePackages {
                         importFrom("BuildXL.Utilities.Instrumentation").withQualifier(net451Qualifier).Common.dll.runtime,
                         ...importFrom("Microsoft.Diagnostics.Tracing.EventSource.Redist").withQualifier(
                             { targetFramework: "net451" }).pkg.runtime,
-                        ...importFrom("Microsoft.Applications.Telemetry.Desktop").withQualifier(

```

Comment:
We build our utilities targeting net451 for AzDev.  How much do we care about all the tracing stuff working correctly there? (i.e., is that even being used there?)  .NET Framework 4.5.1 doesn't have all the tracing features we use.  So can we either fake it with `#if NET_FRAMEWORK_451` (like in this PR) or simply change the target 4.5.1 to 4.6.2?

Reply:
I suspect for AzDev's usage ETW tracing is largely a side effect and not observed. @dcwuser would know best. David - do you think this change is big enough to necessitate getting that ingestion process fixed again?

Reply:
Isn't that ingestion already broken?  I don't remember when was the last time it passed in our RM validation